### PR TITLE
Merge pull request #6481 from nicoddemus/release-5.3.3

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

File: doc/en/changelog.rst [hunk 1]
```diff
@@ -4842,7 +4870,7 @@ time or change existing behaviors in order to make them less surprising/more use
 * Updated docstrings with a more uniform style.
 
 * Add stderr write for ``pytest.exit(msg)`` during startup. Previously the message was never shown.
-  Thanks `@BeyondEvil`_ for reporting `#1210`_. Thanks to `@jgsonesen`_ and
+  Thanks `@BeyondEvil`_ for reporting `#1210`_. Thanks to @jgsonesen and
```
Comment (nicoddemus) [inline doc/en/changelog.rst]: @jgsonesen is a broken link

Comment (blueyed) [reply doc/en/changelog.rst]: For reference: was even changed before apparently already.  Original PR: https://github.com/pytest-dev/pytest/pull/1763.

File: doc/en/changelog.rst [hunk 2]
```diff
@@ -28,6 +28,34 @@ with advance notice in the **Deprecations** section of releases.
 
 .. towncrier release notes start
 
+pytest 5.3.3 (2020-01-16)
+=========================
+
+Bug Fixes
```
Comment (blueyed) [inline doc/en/changelog.rst]: It's been like that forever, but wouldn't "Bugfixes" be more common?

File: doc/en/announce/release-5.3.3.rst [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+Thanks to all who contributed to this release, among them:
+
+* Adam Johnson
+* Alexandre Mulatinho
+* Anthony Sottile
+* Bruno Oliveira
+* Chris NeJame
+* Daniel Hahler
+* Hugo
```
Comment (hugovk) [inline doc/en/announce/release-5.3.3.rst]:
```suggestion
* Hugo van Kemenade
```

File: doc/en/changelog.rst (hunk 2 above)
Comment (hugovk) [reply doc/en/changelog.rst]: No, "Bug fixes" is more common with 6,063,858 available code results:

https://github.com/search?utf8=%E2%9C%93&q=%22Bug+fixes%22&type=Code

"Bugfix" has 3,876,434:

https://github.com/search?q=Bugfixes&type=Code

File: doc/en/announce/release-5.3.3.rst (hunk 3 above)
Comment (blueyed) [reply doc/en/announce/release-5.3.3.rst]: @hugovk 
You might want to fix/adjust your Git config then maybe (it comes via 8de9b1b).

File: doc/en/changelog.rst (hunk 1 above)
Comment (nicoddemus) [reply doc/en/changelog.rst]: 👍

File: doc/en/changelog.rst (hunk 2 above)
Comment (blueyed) [reply doc/en/changelog.rst]: => https://github.com/pytest-dev/pytest/pull/6549#discussion_r370273926
As per Google "bugfixes" is the more popular one.

**Pull request merged**